variables.tf

```diff
@@ -151,6 +151,26 @@ variable "se_instance_type" {
   type        = string
   default     = "c5.large"
 }
+variable "se_s3_encryption" {
+  description = "Enable encryption on SE S3 Bucket"
```

Comment:
could you add something about the default behavior is generate new KMS keys unless the associated key_arn variable is set to true?

Reply:
Absolutely